src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultModelBindingContext.cs

```diff
@@ -130,32 +132,16 @@ public DefaultModelBindingContext()
         }
 
         /// <inheritdoc />
-        public override NestedScope EnterNestedScope()
```

Comment:
1. You split these methods from the other `EnterNestedScope()` overload.
2. There is no such guideline; no mention of member order in https://github.com/aspnet/Home/wiki/Engineering-guidelines#coding-guidelines. It's an informal policy we've imposed on new code and not something we rewrite classes to match.